apps/docs/src/.vitepress/components/IconLibraryItem.vue

```diff
@@ -17,16 +18,19 @@ const handleCopy = async () => {
   isCopied.value = true;
   setTimeout(() => (isCopied.value = false), 3000);
 };
+
+const tooltipText = computed(() => {
+  if (!isCopied.value) return props.icon.tooltipName;
+  return `${props.icon.tooltipName} - Import copied to clipboard!`;
+});
 </script>
 
 <template>
-  <button type="button" class="icon" @click="handleCopy">
-    <OnyxIcon :icon="props.icon.content" />
-    <span class="icon__tooltip">
-      {{ props.icon.tooltipName }}
-      <div v-if="isCopied">Import copy successful!</div>
-    </span>
-  </button>
+  <OnyxTooltip :text="tooltipText">
```

Comment:
just a tiny nice-to-have idea 😋: we could show a "check" icon when the copy successful text appears :)